### DISCUSSION TITLE
Testing custom device configs in HassOS

### DISCUSSION THREAD

**Discussion by disforw** [zwave-js/zwave-js#2848] [Q&A] [answered]
I recently learned about this awesome ability to mount and load custom config files in the store/config directory. Having a Home Assistant Blue (and therefore HassOS), I do not have access to the store folder or the ZwaveJS file structure at all. We only have access to the Home Assistant config folder. I cant easily access the file to change “deviceConfigPriorityDir”. Has anyone found a way around this?

**Comment by AlCalzone** [comment]
I'll ask the HA devs if they can provide some input here.

**Comment by kpine** [comment, marked as answer]
Not possible until the add-on infrastructure supports it. https://github.com/home-assistant/supervisor/issues/1842

There are advanced ways such as entering the container via command lines, or enabling debugging modes, but these are not reasonable for normal users.

**Reply by kpine** [reply]
I should also mention that neither of the add-ons (official or community zwavejs2mqtt) set the `deviceConfigPriorityDir` in the configuration either, so you'd still have to modify files in the default location if you tried to manually access the container. Assuming that RFC is ever implemented, I'd expect the add-ons to eventually support it.

**Reply by glyph-se** [reply]
zwavejs2mqtt sets it now: https://github.com/zwave-js/zwavejs2mqtt/pull/1294
